unit-tests/src/test/java/org/eclipse/collections/impl/factory/ListsTest.java

```diff
@@ -185,6 +185,24 @@ public void multiReader()
         Verify.assertInstanceOf(MultiReaderList.class, listFactory.fromStream(Stream.of(1, 2, 3)));
     }
 
+    @Test
+    public void apiMultiReader()
```

Comment:
I would remove @Test annotation from this method declare it as a private void testMultiReaderApi(MultiReaderListFactory listFactory). Since listFactory is now passed as a parameter, I would remove line 191. Everything else looks great in the method. Hence, can stay.

Reply:
done!